services/resource-usage-tracker/Makefile

```diff
@@ -0,0 +1,18 @@
+#
+# DEVELOPMENT recipes for resource-usage-tracker
+#
+include ../../scripts/common.Makefile
+include ../../scripts/common-service.Makefile
+
+.env:
+	$(APP_CLI_NAME) generate-dotenv > $@
```

Comment:
I thought you told me you would remove this CLI command! :-)

Reply:
yes I did. I forgot that thing